packages/vite/bins/rw-vite-dev.mjs

```diff
@@ -0,0 +1,31 @@
+import { createServer } from 'vite'
+
+import { getPaths } from '@redwoodjs/internal/dist/paths.js'
+
+const rwPaths = getPaths()
+
+const startDevServer = async () => {
+  const configFile = rwPaths.web.viteConfig
+
+  if (!configFile) {
+    throw new Error('Could not locate your web/vite.config.{js,ts} file')
+  }
+
+  const devServer = await createServer({
+    configFile,
+    envFile: false, // env file is handled by plugins in the redwood-vite plugin
+  })
+
+  await devServer.listen()
+
+  process.stdin.on('data', (data) => {
+    const str = data.toString().trim().toLowerCase()
+    if (str === 'rs' || str === 'restart') {
+      devServer.restart(true)
+    }
+  })
+
+  devServer.printUrls()
+}
+
+startDevServer()
```

Comment:
Is there a reason we need this bin still if you've made it all programatic? I thought that was the original endeavour, to get rid of this bin file and move all this code to `yarn rw dev`, like you did with `buildWeb` in this PR.

Reply:
Concurrently, which we use in dev, only works with cli processes as far as I can see!

Reply:
Ah yeah that makes sense